#Servico para parar pose do NAO
---
bool robot_has_fallen
